src/components/Project/index.tsx

```diff
@@ -48,15 +59,26 @@ const getCategoryUnit = (
 export const Project: FC<ProjectsType> = project => {
   const [selectedDeviceIndex, setSelectedDeviceIndex] = useState<number>(0);
   const selectedDevice = project?.devices?.[selectedDeviceIndex];
+  const [currentDatetimeRange, setCurrentDatetimeRange] = useState<{
+    startDateTimeString: string | undefined;
+    endDateTimeString: string | undefined;
+  }>({
+    startDateTimeString: today.toISOString(),
+    endDateTimeString: tenDaysAgo.toISOString(),
```

Comment:
Is it possible that the values are mixed up here? In its default state the filter seems to show the more recent date first and the `tenDaysAgo` afterwards:

![Screenshot 2021-08-18 at 12 14 52](https://user-images.githubusercontent.com/15640196/129881400-2d8cbf11-ee79-460e-8992-c5d445ae9e0e.png)

Or I am misinterpreting that?

Reply:
Good catch! I totally missed that! Thx!